rsmtool/notebooks/header.ipynb

```diff
@@ -350,6 +352,8 @@
     "    if exists(feature_file):\n",
     "        df_features = DataReader.read_from_file(feature_file, converters=converter_dict)\n",
     "        features_used = [c for c in df_features.feature.values]\n",
+    "        # compute the longest feature name: we'll need if for the plots\n",
+    "        longest_feature_name = max(map(len, features_used))\n",
```

Comment:
What if there's no `feature_file`? Doesn't that mean the `longest_feature_name` variable won't exist?

Reply:
That's a good question: we have the check because the header is shared across tools ('contexts') and for example feature_file is not present in RSMEval outputs. 

As things stand now, the feature file will always be present in an RSMTool experiment which is the only context where we would also include the feature analysis notebooks. The definition of `features_used` variable in l.352 follows the same logic.

Reply:
Got it. Thanks! :)